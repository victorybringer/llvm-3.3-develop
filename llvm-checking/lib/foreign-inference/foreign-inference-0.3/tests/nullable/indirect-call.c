#include "base.h"

typedef int (*NullAccessor)(int*, int*);

NullAccessor A = argOneNotNull;

int neitherNullable(int* a, int* b) {
  return *a + *b;
}


void setup() {
  A = neitherNullable;
}

void f(int * a, int * b) {
  A(a, b);
}
