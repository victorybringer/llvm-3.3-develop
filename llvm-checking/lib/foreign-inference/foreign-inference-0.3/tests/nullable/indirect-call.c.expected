fromList [ ("@neitherNullable", fromList ["%a", "%b"])
         ]
